integration/pull_test.go

```diff
@@ -501,6 +504,148 @@ level = "debug"
 	}
 }
 
+// TestPullWithArbitraryBlobAsZtoc tests the snapshotter behavior if an arbitrary blob is passed
```

Comment:
should we highlight "an arbitrary blob _with invalid ztoc format_"? Since in the future we may add another test for "an arbitrary blob _with valid ztoc format but invalid data_".